src/main/java/com/iota/iri/network/pipeline/SolidifyPayload.java

```diff
@@ -0,0 +1,24 @@
+package com.iota.iri.network.pipeline;
+
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.network.neighbor.Neighbor;
+
+public class SolidifyPayload extends Payload {
+    Neighbor originNeighbor;
+    TransactionViewModel tvm;
+
+    public SolidifyPayload(Neighbor originNeighbor, TransactionViewModel tvm){
+        this.originNeighbor = originNeighbor;
+        this.tvm = tvm;
+    }
+
+
+    @Override
+    public Neighbor getOriginNeighbor(){
+        return originNeighbor;
+    }
+
+    public TransactionViewModel getTransaction(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4339340)